python/kserve/kserve/protocol/rest/server.py

```diff
@@ -174,7 +173,7 @@ def create_application(self) -> FastAPI:
                     r"/v2/models/{model_name}/infer",
                     v2_endpoints.infer,
                     methods=["POST"],
-                    response_model=InferenceResponse,
+                    response_model=None,
```

Comment:
The downside of this is that we lose the validation when returning back the `InferenceResponse`

Reply:
The response will be validated [here](https://github.com/kserve/kserve/pull/3685/files#diff-3aa3dd39255bdc2c68b576b8abdc038a614324fd09542b3d137ac9c28e0d8f2bR180)